### PR TITLE
Misspell correction

### PR DESCRIPTION
**Pull request by yongtang**
Did a `misspell . | grep -v ^vendor` and fixed several typos.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: core/zmiddleware.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	_ "github.com/coredns/coredns/middleware/chaos"
 	_ "github.com/coredns/coredns/middleware/debug"
 	_ "github.com/coredns/coredns/middleware/dnssec"
+	_ "github.com/coredns/coredns/middleware/dnstap"
```
Comment (miekg) [inline core/zmiddleware.go:13]: we didn't enabled it?

Comment (chrisohaver) [reply core/zmiddleware.go:13]: Yeah - this has been really annoying.

Comment (chrisohaver) [reply core/zmiddleware.go:13]: probably included by accident in this PR.  but should be fixed anyway...

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [coredns/coredns] Misspell corr..." ]
>chrisohaver commented on this pull request.
>
>
>
>> @@ -10,6 +10,7 @@ import (
> 	_ "github.com/coredns/coredns/middleware/chaos"
> 	_ "github.com/coredns/coredns/middleware/debug"
> 	_ "github.com/coredns/coredns/middleware/dnssec"
>+	_ "github.com/coredns/coredns/middleware/dnstap"
>
>probably included by accident in this PR.  but should be fixed anyway...

Indeed. The release-build Makefile doesn't include a 'go generate' step, it 
probably should.

**Comment by yongtang** [comment]
I did a `make` before push the PR. Might be generated automatically then.

**Comment by johnbelamaric** [comment]
yeah, the regular make does that, but not the release make.

**Comment by yongtang** [comment]
It would be good if we could consolidate Makefile and Makefile.release so that they have the same flow and flags.

**Pull request merged**